pipeline/git/git.go

```diff
@@ -30,5 +34,7 @@ func (Pipe) Run(ctx *context.Context) (err error) {
 		PreviousTag: previous,
 		Diff:        log,
 	}
+	// removes usual `v` prefix
+	ctx.Version = strings.TrimPrefix(tag, "v")
```

Comment:
Seems like homebrew also ideally expects semantic versioning too (`^\d+\.\d+\.\d+`).

I'll just point that out in the README.